js/common.js

```diff
@@ -1,3 +1,25 @@
+//Mobile navigation toggle
+$(document).ready(function(){
+    $('#nav-icon-menu').click(function(){
+        $(this).toggleClass('open');
+        $('body').toggleClass('navigation-opened');
+        return false;
+    });
+});
+
+//Mobile navigation — doc-list toggle
+$('.doc-list-toggle').on('click', function(){
+    $('.doc-list-inside').toggleClass('active');
+    $(this).toggleClass('active');
+});
+
+//Mobile footer navitation
+$(document).ready(function() {
+    var $window = $(window);
+    $('.toggle').on('click',function(){
```

Comment:
Move this part of code to `$(document).ready(function() {}`  upper.  We already have this event.